index.bs

```diff
@@ -11288,7 +11288,16 @@ with the [{{LegacyNoInterfaceObject}}] [=extended attribute=].
         then set |proto| to |realm|.\[[Intrinsics]].[[{{%Error.prototype%}}]].
     1.  Otherwise, set |proto| to |realm|.\[[Intrinsics]].[[{{%Object.prototype%}}]].
     1.  Assert: <a abstract-op>Type</a>(|proto|) is Object.
-    1.  Let |interfaceProtoObj| be [=!=] [$OrdinaryObjectCreate$](|proto|).
+    1.  If |interface| is declared with the [{{Global}}] [=extended attribute=], or
+        |interface| is in the set of [=inherited interfaces=] of an interface
+        that is declared with the [{{Global}}] [=extended attribute=], then:
+        1.  Let |interfaceProtoObj| be [=!=] [$MakeBasicObject$](« \[[Prototype]], \[[Extensible]] »).
```

Comment:
This declares a variable which immediately goes "out of scope". The "Let" needs to be earlier. (Perhaps by moving the "Let interfaceProtoObj be OrdinaryObjectCreate(proto)" before, and only overriding it when the if condition is true.)

Reply:
This differs from **ECMA262**, where the step “Let <var>interfaceProtoObj</var> be ! OrdinaryObjectCreate(proto)” creates a hoisted variable <var>interfaceProtoObj</var>.